parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs

```diff
@@ -1122,26 +1168,46 @@ mod tests {
 			frame_system::CheckSpecVersion::new(),
 			frame_system::CheckTxVersion::new(),
 			frame_system::CheckGenesis::new(),
-			frame_system::CheckEra::from(Era::Immortal),
+			frame_system::CheckEra::from(sp_runtime::generic::Era::Immortal),
 			frame_system::CheckNonce::from(10),
 			frame_system::CheckWeight::new(),
 			pallet_transaction_payment::ChargeTransactionPayment::from(10),
 			BridgeRejectObsoleteHeadersAndMessages {},
+			(
+				BridgeRefundBridgeHubRococoMessages::default(),
+				BridgeRefundBridgeHubWococoMessages::default(),
+			),
 		);
 
-		let bhr_indirect_payload = bp_bridge_hub_rococo::SignedExtension::new(
-			((), (), (), (), Era::Immortal, 10.into(), (), 10.into(), ()),
-			None,
-		);
+		let bhr_indirect_payload =
+			bp_bridge_hub_rococo::rewarding_bridge_signed_extension::from_params(
+				10,
+				10,
+				TransactionEra::Immortal,
+				test_header::<bridge_hub_test_utils::RelayBlockHeader>(1).hash(),
+				10,
+				10,
+			);
 		assert_eq!(payload.encode(), bhr_indirect_payload.encode());
 
-		let bhw_indirect_payload = bp_bridge_hub_wococo::SignedExtension::new(
-			((), (), (), (), Era::Immortal, 10.into(), (), 10.into(), ()),
-			None,
-		);
+		let bhw_indirect_payload =
+			bp_bridge_hub_wococo::rewarding_bridge_signed_extension::from_params(
+				10,
+				10,
+				TransactionEra::Immortal,
+				test_header::<bridge_hub_test_utils::RelayBlockHeader>(1).hash(),
+				10,
+				10,
+			);
 		assert_eq!(payload.encode(), bhw_indirect_payload.encode());
 
-		check_additional_signed::<SignedExtra, bp_bridge_hub_rococo::SignedExtension>();
-		check_additional_signed::<SignedExtra, bp_bridge_hub_wococo::SignedExtension>();
+		check_additional_signed::<
```

Comment:
@serban300 
the last thing, this `check_additional_signed` does not compile now, 
because `()` vs
`(BridgeRefundBridgeHubRococoMessages::default(), BridgeRefundBridgeHubWococoMessages::default())`

what do you suggest?
rewrite check_additional_signed to check data instead of compilation?

Reply:
Oh, I forgot about it. 

> what do you suggest?
rewrite check_additional_signed to check data instead of compilation?

Yes, probably.